### DISCUSSION TITLE
Sort selected lines of code in alphabatical order.

### DISCUSSION THREAD

**Discussion by grv07** [helix-editor/helix#2866] [Q&A] [answered]
I am new in helix, I like to have some keybindings to sort the selected lines in alphabetical order. 

I try to run ":sort" command after selecting some lines in my code editor but it did not do anything.

Is there already any keybinding available for this?

**Comment by ChrHorn** [comment, marked as answer]
On Unix or Powershell/Cmd you can use `:pipe sort`.

**Reply by the-mikedavis** [reply]
As a keybind for this there is `|`. `|sort` to sort a multi-line selection

**Comment by the-mikedavis** [comment]
`:sort` works on mulitple selections rather than a single multi-line selection. You can split a multi-line selection into multiple selections linewise with <kbd>Alt</kbd><kbd>s</kbd>
